tests/db/test_orm.py

```diff
@@ -2,7 +2,9 @@
 import unittest
 
 from remoteappmanager.db import orm
-from remoteappmanager.db.orm import Database, transaction, Accounting
+from remoteappmanager.db.orm import (Database, transaction, Accounting,
+                                     AccountingInterface)
```

Comment:
(suggestion): I tend to not name concrete implementations as `Interface`. how about `UserAccounting`